ddl/placement/types.go

```diff
@@ -94,7 +46,7 @@ type Rule struct {
 	EndKeyHex        string            `json:"end_key"`
 	Role             PeerRoleType      `json:"role"`
 	Count            int               `json:"count"`
-	LabelConstraints []LabelConstraint `json:"label_constraints,omitempty"`
+	LabelConstraints []Constraint `json:"label_constraints,omitempty"`
```

Comment:
The linter is complaining this file needs a go fmt run. I suspect it is this line changing indentation.

Reply:
DONE